src/core/Akka/Util/AtomicBoolean.cs

```diff
@@ -70,30 +70,6 @@ public bool GetAndSet(bool newValue)
         {
             return Interlocked.Exchange(ref _value, newValue ? _trueValue : _falseValue) == _trueValue;
         }
-
-        #region Conversion operators
-
-        /// <summary>
-        /// Performs an implicit conversion from <see cref="AtomicBoolean"/> to <see cref="System.Boolean"/>.
-        /// </summary>
-        /// <param name="atomicBoolean">The boolean to convert</param>
-        /// <returns>The result of the conversion.</returns>
-        public static implicit operator bool(AtomicBoolean atomicBoolean)
-        {
-            return atomicBoolean.Value;
```

Comment:
Shouldn't this be fine?

Reply:
Yeah I guess so - that's probably safe. I can add that one back.

Reply:
Done